### ISSUE TITLE
E-mails not being delivered to verizon.net domain

### ISSUE THREAD

**Issue by bzark** [mail-in-a-box/mailinabox#1226]
New install of mail-in-a-box and when I try to send to my verizon.net account it spits back the error below.  Earlier this year AOL acquired verizon.net e-mail services so I suspect this has something to do with it.  I omitted the e-mail username for security reasons.

This is the mail system at host box.zanypixels.com.

####################################################################
# THIS IS A WARNING ONLY.  YOU DO NOT NEED TO RESEND YOUR MESSAGE. #
####################################################################

Your message could not be delivered for more than 3 hour(s).
It will be retried until it is 2 day(s) old.

For further assistance, please send mail to postmaster.

If you do so, please include this problem report. You can
delete your own text from the attached returned message.

                   The mail system

<xxxxx@verizon.net>: connect to mailin-04.mx.aol.com[152.163.0.100]:25:
    Connection timed out

**details.txt**

Reporting-MTA: dns; box.zanypixels.com
X-Postfix-Queue-ID: B32478330C
X-Postfix-Sender: rfc822; xxxx@zanypixels.com
Arrival-Date: Mon, 31 Jul 2017 00:47:07 -0400 (EDT)

Final-Recipient: rfc822; xxxxx@verizon.net
Original-Recipient: rfc822;xxxxx@verizon.net
Action: delayed
Status: 4.4.1
Diagnostic-Code: X-Postfix; connect to mailin-04.mx.aol.com[152.163.0.100]:25:
    Connection timed out
Will-Retry-Until: Wed,  2 Aug 2017 00:47:07 -0400 (EDT)

**Undelivered Message Header.txt**

Return-Path: <xxxx@zanypixels.com>
Received: from authenticated-user (box.zanypixels.com [52.0.18.55])
	(using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
	(No client certificate requested)
	by box.zanypixels.com (Postfix) with ESMTPSA id B32478330C
	for <xxxxx@verizon.net>; Mon, 31 Jul 2017 00:47:07 -0400 (EDT)
DKIM-Signature: v=1; a=rsa-sha256; c=simple/simple; d=zanypixels.com;
	s=mail; t=1501476427;
	bh=ETLoPz9U8M288Uz3iVPZRB19V4w+XFk8qDvgblJ53PY=;
	h=Date:Subject:To:From:From;
	b=Uz3VkmEpJ49VWvXgNfTKMoHoYaMRuLAwseCiXiutsYAv7xmGrQzAydxJmpmwbll7j
	 Hm3ZDXJR6DQakHDhzIL/2+9L+T0jFHzfI/hafwxBX+DtX2u4r1KTnS54TFc9srBCcA
	 bamebBGJzru3v9cApTpS50g18Lee9wNSduWFAiwqg/zxytCRDMsp0RJm30D7iuAE7A
	 scZGtjs6VO3KMcyWaLc/aC45ISQTOsDHRwBuik9l5u/hBiM2XQBRdU40gl9ke0oJAg
	 XeD0qAiaBhHnf8PFw/fOEUFyasrTOn+/UHgFmexTCYhuLxbHvNfovieOpkLQsc8BNI
	 YPxmOJ8yjtbIA==
Date: Mon, 31 Jul 2017 00:47:08 -0400
MIME-Version: 1.0
Content-Type: multipart/alternative; boundary="----0Q5GTY8WO1KHXIJMNTDGIW52GVVRIV"
Content-Transfer-Encoding: 7bit
Subject: Test
To: Bret <xxxxx@verizon.net>
From: Bret <xxxx@zanypixels.com>
Message-ID: <B8A6CAD6-928D-4949-BA39-04DF5243D9EE@zanypixels.com>

**Comment by bzark** [comment]
Also I tried to sign up to the forum to report this issue but I believe it is the same issue because I never received the e-mail to confirm my account.

**Comment by bzark** [comment]
This had to do with Amazon AWS not setting up reverse DNS.  They did that last night and the e-mails were delivered.

**Comment by bzark** [comment]
So as my e-mail server is working okay with verizon.net e-mail addresses, I don't believe yours is.  I have resent the confirmation e-mail for the forum several times and still haven't received it (not in SPAM either).

**Comment by damongant** [comment]
https://mailinabox.email/guide.html#machine

>(Most any cloud provider will do, but not Amazon Web Services because its network is often blocked to prevent users from sending spam.)

**Comment by bzark** [comment]
@damongant your wrong, don't believe old policies.  Do your research and you will see it is completely acceptable to run a mail server using AWS.  That should be removed from the setup guide.  I have been running two separate boxes in AWS, totaling 5 domains and I have zero issues.  KUDOS to MIAB and Josh.

Like I previously said, after submitting my reverse DNS to AWS all queued e-mails and all future e-mails have been successfully delivered.

**Comment by damongant** [comment]
The policy is still in place. Yes, you can run a MTA on AWS, but you need [their prior approval](https://aws.amazon.com/premiumsupport/knowledge-center/ec2-port-25-throttle/) if you don't want to get tarpit'd. Your issue is consistent with not being whitelisted, being a timeout.

**Comment by bzark** [comment]
My prior issue was with not setting up reverse DNS which can simply be done using this link and then your MIAB runs perfectly.  So once again NOT an AWS issue.

[https://aws-portal.amazon.com/gp/aws/html-forms-controller/contactus/ec2-email-limit-rdns-request](url)

Once that was completed I had no issues with verizon.net e-mails.  Prior to setting that up I had no issues with any other domains including Gmail.  I clearly stated what needed to be done in this Git issue, not sure what you are trying to do other than argue.  My boxes are working fine.

I only re-commented on the issue to bring to Josh's attention that when using a verizon.net e-mail address when signing up to the forum, the e-mails were not being delivered.  So if Josh is using AWS then that may be the issue, if not, it could be another issue.  Earlier this year Verizon sold off their e-mail division to AOL.  I originally thought that my have been the issue.

**Comment by JoshData** [comment]
My personal Mail-in-a-Box has long been blocked by AOL (I can't email my parents!) and I think the Verizon acquisition is about when that block started too. Unfortunately I probably can't fix this. AOL Postmaster info has not been helpful.

**Comment by bzark** [comment]
@JoshData thanks for reporting.  I have no issues using MIAB with AOL or verizon.net e-mail addresses.  I'm sure you checked that your IP address is not blacklisted.  BTW, who is hosting your MIAB?

**Comment by JoshData** [comment]
My IP address is blacklisted by AOL (and no one else). The box is at Rimuhosting (but it's not their fault, they're good).

**Comment by bzark** [comment]
Oh, so then the issue is known for your MIAB.  As big as AOL and AWS are, I'm sure they have some inside relationships that go beyond "policies."  You can go ahead and close this issue then.

And THANK YOU for all your hard work with MIAB.  I have to say this is one sweet "box."

**Comment by JoshData** [comment]
Thanks (and on behalf of all of the contributors). :)
